internal/v1/api.yaml

```diff
@@ -207,6 +207,15 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPErrorList'
+    patch:
+      summary: Apply linter fixes to blueprint
+      description: |
+        Apply fixes which should fix any lint errors in the blueprint.
+      responses:
+        200:
+          description: successful update
+        404:
+          description: blueprint was not found
```

Comment:
Not fan of using HTTP PATCH on the blueprint resource for this, I encourage you to introduce a new sub-resource for this.